src/tests/ftest/datamover/large_file.py

```diff
@@ -36,52 +36,51 @@ def run_dm_large_file(self, tool):
         self.set_tool(tool)
 
         # Get the number of ior processes
-        self.ior_processes = self.params.get(
-            self.tool.lower(), "/run/ior/client_processes/*")
+        self.ior_processes = self.params.get(self.tool.lower(), "/run/ior/client_processes/*")
         if not self.ior_processes:
             self.fail("Failed to get ior processes for {}".format(self.tool))
 
         # create pool and cont
-        self.create_pool()
-        self.create_cont(self.pool[0])
+        pool = self.create_pool()
+        cont1 = self.create_cont(pool)
 
         # update and run ior on cont1
         self.run_ior_with_params(
             "DAOS", self.ior_cmd.test_file.value,
-            self.pool[0], self.container[0])
+            pool, self.container[0])
```

Comment:
Did this get missed?
```suggestion
            pool, cont1])
```